libtiledbsoma/src/soma/soma_array.h

```diff
@@ -77,6 +77,7 @@ class SOMAArray {
      * @param batch_size Read batch size
      * @param result_order Read result order: automatic (default), rowmajor, or
      * colmajor
+     * @param timestamp Pair indicating timestamp start and end
```

Comment:
Should we say 'optional'?  Don't we usually signal when a parameter is not required?